operator/controllers/common/taskhandler.go

```diff
@@ -33,7 +34,7 @@ type TaskCreateAttributes struct {
 	CheckType      common.CheckType
 }
 
-func (r TaskHandler) ReconcileTasks(ctx context.Context, reconcileObject client.Object, taskCreateAttributes TaskCreateAttributes) ([]klcv1alpha1.TaskStatus, apicommon.StatusSummary, error) {
+func (r TaskHandler) ReconcileTasks(ctx context.Context, phaseCtx context.Context, reconcileObject client.Object, taskCreateAttributes TaskCreateAttributes) ([]klcv1alpha1.TaskStatus, apicommon.StatusSummary, error) {
```

Comment:
can we add a unit test for this function where we also check if the spanHandler is used as expected (i.e. creating and closing the span)?

Reply:
added